R/variable.R

```diff
@@ -23,6 +23,11 @@ setMethod("name<-", "CrunchVariable",
     function (x, value) setTupleSlot(x, "name", validateNewName(value)))
 #' @rdname describe
 #' @export
+setMethod("id", "CrunchVariable", function (x) {
+    return(x@tuple@body$id)
+})
```

Comment:
I'm glad we have this now! I have totally called it expecting it to work and then been reminded "oh right I should go add that...."

Reply:
The way to define this method that is consistent with the rest above and below it is `tuple(x)$id`.